modules/globebrowsing/globebrowsingmodule_lua.inl

```diff
@@ -151,32 +162,40 @@ namespace {
     return res;
 }
 
-/**
- * Returns the list of layers for the scene graph node specified in the first parameter.
- * The second parameter specifies which layer type should be queried. Deprecated in favor
- * of 'layers'.
- */
+ /**
+  * Returns the list of layers for the specified globe for a specific layer group.
+  *
+  * Deprecated in favor of `layers`.
+  *
+  * \param globeIdentifier The identifier of the scene graph node for the globe
+  * \param layerGroupName The name of the layer group for which to list the layers
+  */
 [[codegen::luawrap("getLayers")]] std::vector<std::string> layersDeprecated(
                                                               std::string globeIdentifier,
-                                                                        std::string layer)
+                                                               std::string layerGroupName)
 {
     LWARNINGC(
         "Deprecation",
         "'getLayers' function is deprecated and should be replaced with 'layers'"
     );
 
-    return layers(std::move(globeIdentifier), std::move(layer));
+    return layers(std::move(globeIdentifier), std::move(layerGroupName));
 }
 
 /**
- * Rearranges the order of a single layer on a globe. The first parameter is the
- * identifier of the globe, the second parameter specifies the name of the layer group,
- * the third parameter is the original position of the layer that should be moved and the
- * last parameter is the new position in the list. The third and fourth parameters can
- * also acccept names, in which case these refer to identifiers of the layer to be moved.
- * If the last parameter is a name, the source layer is moved below that destination
- * layer. The first position in the list has index 0, and the last position is given by
- * the number of layers minus one.
+ * Rearranges the order of a single layer on a globe. The first position in the list
+ * has index 0, and the last position is given by the number of layers minus one.
+ *
+ * The `source` and `desitnation` parameters can also acccept names, in which case these
```

Comment:
```suggestion
 * The `source` and `destination` parameters can also accept names, in which case these
```

Reply:
Instead of saying "names" here, maybe already call them identifier?  Something along:
"The source and destination parameters can also be the identifiers of the layers to be moved"?